proxy/src/control/mod.rs

```diff
@@ -115,16 +109,16 @@ impl Background {
 //TODO: move to tower-backoff
 struct Backoff<S> {
     inner: S,
-    timer: ReactorTimeout,
+    timer: Delay,
     waiting: bool,
     wait_dur: Duration,
 }
 
 impl<S> Backoff<S> {
-    fn new(inner: S, wait_dur: Duration, handle: &Handle) -> Self {
+    fn new(inner: S, wait_dur: Duration) -> Self {
         Backoff {
             inner,
-            timer: ReactorTimeout::new(wait_dur, handle).unwrap(),
+            timer: Delay::new(Instant::now() + wait_dur),
```

Comment:
really, all instances of `Instance::now` should be replaced with a time source that can be mocked. I don't think that we should not do that in this PR, but can you open a followup issue for this?

Reply:
nm, https://github.com/runconduit/conduit/issues/455 suffices